ui/src/app/applications/components/application-details/application-details.tsx

```diff
@@ -628,7 +650,14 @@ export class ApplicationDetails extends React.Component<RouteComponentProps<{app
             },
             {
                 iconClassName: 'fa fa-sync',
-                title: <ActionMenuItem actionLabel='Sync' />,
+                title: resourceActions.length ? (
```

Comment:
If I get rid of the conditional here, that is when I get [the error](https://github.com/argoproj/argo-cd/pull/10405#issuecomment-1228618127). Otherwise, there is never a drop down menu.